lib/github-package.js

```diff
@@ -71,6 +72,7 @@ export default class GithubPackage {
           this.setActiveContext(WorkdirContext.absent());
         }
       }),
+      ContextMenuInterceptor,
```

Comment:
I gave the simplification here a shot and our test suite leaked a _ton_ of event listeners, because Enzyme doesn't trigger `componentWillUnmount`. I think I'm just going to leave it as -is with the static Map.